paddle/operators/fill_zeros_like_op.cu

```diff
@@ -0,0 +1,6 @@
+#include "paddle/framework/op_registry.h"
+#include "paddle/operators/fill_zeros_like_op.h"
+
+REGISTER_OP_GPU_KERNEL(
+    fill_zeros_like,
```

Comment:
why name this `fill_zerors_like`, numpy call it [zeros_like](https://docs.scipy.org/doc/numpy/reference/generated/numpy.zeros_like.html)